pygmt/datasets/load_earth_dataset.py

```diff
@@ -0,0 +1,202 @@
+    ----------
+    title : str
+        The title of the dataset, used in error messages.
+
+    name : str
+        The name assigned as an attribute to the DataArray.
+
+    long_name : str
+        The long name assigned as an attribute to the DataArray.
+
+    units : str
+        The units of the values in the DataArray.
+
+    resolutions : dict
+        Dictionary of available resolution as keys and the values are
+        Resolution objects.
+
+    attribute_datum : dict
+        A dictionary of the datum or data for the axis on the dataset.
+    """
+
+    title: str
+    name: str
+    long_name: str
+    units: str
+    resolutions: Dict[str, Resolution]
+    attribute_datum: dict
+
+
+datasets = {
+    "earth_relief": GMTRemoteDataset(
+        title="Earth relief",
+        name="elevation",
+        long_name="Earth elevation relative to the geoid",
+        units="meters",
+        attribute_datum={"vertical_datum": "EMG96", "horizontal_datum": "WGS84"},
+        resolutions={
+            "01d": Resolution(["pixel", "gridline"], False),
+            "30m": Resolution(["pixel", "gridline"], False),
+            "20m": Resolution(["pixel", "gridline"], False),
+            "15m": Resolution(["pixel", "gridline"], False),
+            "10m": Resolution(["pixel", "gridline"], False),
+            "06m": Resolution(["pixel", "gridline"], False),
+            "05m": Resolution(["pixel", "gridline"], True),
+            "04m": Resolution(["pixel", "gridline"], True),
+            "03m": Resolution(["pixel", "gridline"], True),
+            "02m": Resolution(["pixel", "gridline"], True),
+            "01m": Resolution(["pixel", "gridline"], True),
+            "30s": Resolution(["pixel", "gridline"], True),
+            "15s": Resolution(["pixel"], True),
+            "03s": Resolution(["gridline"], True),
+            "01s": Resolution(["gridline"], True),
+        },
+    ),
+    "earth_age": GMTRemoteDataset(
+        title="seafloor age",
+        name="seafloor_age",
+        long_name="age of seafloor crust",
+        units="Myr",
+        attribute_datum={"horizontal_datum": "WGS84"},
+        resolutions={
+            "01d": Resolution(["pixel", "gridline"], False),
+            "30m": Resolution(["pixel", "gridline"], False),
+            "20m": Resolution(["pixel", "gridline"], False),
+            "15m": Resolution(["pixel", "gridline"], False),
+            "10m": Resolution(["pixel", "gridline"], False),
+            "06m": Resolution(["pixel", "gridline"], False),
+            "05m": Resolution(["pixel", "gridline"], True),
+            "04m": Resolution(["pixel", "gridline"], True),
+            "03m": Resolution(["pixel", "gridline"], True),
+            "02m": Resolution(["pixel", "gridline"], True),
+            "01m": Resolution(["gridline"], True),
+        },
+    ),
+}
+
+
+@kwargs_to_strings(region="sequence")
+def _load_earth_dataset(dataset_name, dataset_prefix, resolution, region, registration):
```

Comment:
I think we should go with `_load_remote_dataset`:
```suggestion
def _load_remote_dataset(dataset_name, dataset_prefix, resolution, region, registration):
```